spec/models/file_set/file_set_derivatives_spec.rb

```diff
@@ -5,7 +5,7 @@
   let(:file)        { build(:department_file, id: '1234') }
   let(:derivatives) { CurationConcerns::DerivativePath.derivatives_for_reference(file) }
 
-  before { LakeshoreTesting.reset_derivatives }
+  before(:each) { LakeshoreTesting.reset_derivatives }
```

Comment:
I had failing tests without it, with seed 14171, so I added it ("I put LakeshoreTesting.reset_derivatives in a before(:each), clears up data that caused a test to fail when run in a particular order (pdf text extraction after images, seed 14171).")